frontend/src/modules/home/quiz/components/PlayButton.js

```diff
@@ -2,22 +2,25 @@ import { Button } from "@material-ui/core"
 import PlayArrowIcon from "@material-ui/icons/PlayArrow";
 import * as Tone from "tone";
 
-const PlayButton = ({first, second}) => {
-    const playTone = () => {
+
+const playTone = (first, second) => {
         //create a synth and connect it to the main output (your speakers)
         const synth = new Tone.Synth().toDestination();
         const now = Tone.now()
 
         //play a middle 'C' for the duration of an 8th note
         synth.triggerAttackRelease(first, "8n", now);
         synth.triggerAttackRelease(second, "8n", now + 0.5);
-    }
+}
+
+const PlayButton = ({first, second}) => {
 
     return (
-        <Button onClick={playTone} color="primary">
+        <Button onClick={() => {playTone(first, second)}} color="primary">
             <PlayArrowIcon />
         </Button>
     )
 }
 
-export default PlayButton;
+export {playTone};
+export default PlayButton;
```

Comment:
Add a newline to the end of the file
```suggestion
export default PlayButton;

```